# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/layers/ipc/LayersSurfaces.ipdlh
# Commit: a8ce70eff417
# Full Hash: a8ce70eff41744ce26905ea160b21b63516c461b
# Author: stransky <stransky@redhat.com>
# Date: 2025-01-08 04:42:02
# Regressor Bug: 1940037
# File Overlap Count: 1
# Description:
#   Bug 1940037 [Wayland] Use exact FOURCC format for DMABuf surfaces r=emilio
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D233241
# ==============================================================================

diff -r 20a5670ea914 -r a8ce70eff417 gfx/layers/ipc/LayersSurfaces.ipdlh
--- a/gfx/layers/ipc/LayersSurfaces.ipdlh	Tue Jan 07 11:58:50 2025 +0000
+++ b/gfx/layers/ipc/LayersSurfaces.ipdlh	Tue Jan 07 11:58:51 2025 +0000
@@ -73,6 +73,7 @@
 
 [Comparable] struct SurfaceDescriptorDMABuf {
   uint32_t bufferType;
+  uint32_t fourccFormat;
   uint64_t[] modifier;
   uint32_t flags;
   FileHandleWrapper[] fds;